doc/source/whatsnew/v0.25.0.rst

```diff
@@ -327,6 +327,7 @@ Indexing
 ^^^^^^^^
 
 - Improved exception message when calling :meth:`DataFrame.iloc` with a list of non-numeric objects (:issue:`25753`).
+- Bug in :meth:`DataFrame.loc` and :meth:`Series.loc` where KeyError was not raised for a ``MultiIndex``. Now, IndexingError takes precedence over KeyError (:issue:`14885`).
```

Comment:
use double backticks on KeyError; use ``:class:`MultiIndex` ``;  maybe something more like

Bug in :meth:`DataFrame.loc` and :meth:`Series.loc` where ``KeyError`` was not raised for a ``MultiIndex`` when the key was longer that the number of levels in the :class:MUutiIndex.  (:issue:`14885`).

Reply:
@jreback actually when key is longer than the levels of the multiindex `IndexingError: Too many indexers` was raised and  is still raised. So would this be more proper?

Bug in :meth:\`DataFrame.loc\` and :meth:\`Series.loc\` where \`\`KeyError\`\` was not raised for a \`\`MultiIndex\`\` when the key was less than or equal to the number of levels in the :class:\`MultiIndex\` (:issue:\`14885\`).